### PR TITLE
Create data model

### PR DESCRIPTION
**Pull request by maciektr**

### CODE REVIEW

File: enroll/fields.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+from django.db import models
+
+DAY_OF_THE_WEEK = {
+    '1': 'Monday',
```
Comment (MrDenkoV) [inline enroll/fields.py:4]: Consider starting indexing with 0.

File: enroll/models.py [hunk 2]
```diff
@@ -1,3 +1,80 @@
...
+
+class Course(models.Model):
+    code = models.CharField(max_length=40)
+    full_name = models.CharField(max_length=100)
+
+    def __str__(self):
+        return self.full_name
+
+
+class ClassTime(models.Model):
```
Comment (MrDenkoV) [inline enroll/models.py:40]: Consider adding frequency of classes - each week, every 2 weeks (A or B), irregular.

**Review by MrDenkoV — COMMENTED** [review]
Great job Mr. Maciek Trątnowiecki!
Consider my comments for possible changes!
Good luck and all the best.
xx

File: enroll/fields.py (hunk 1 above)
Comment (maciektr) [reply enroll/fields.py:4]: Django docs use enumeration types with 1 as the lowest key, e.g. [here](https://docs.djangoproject.com/en/3.1/ref/models/fields/#enumeration-types) (search via IntegerChoices).

Comment (MrDenkoV) [reply enroll/fields.py:4]: Cringe

**Review by MrDenkoV — APPROVED** [review]
Splendid!

**Pull request merged**